lib/swagger/generator.rb

```diff
@@ -28,8 +43,8 @@ def resource_name
       @resource_name ||= collection_name.singularize
     end
 
-    def generate_swagger_schemas
-      name = resource_name
+    def generate_swagger_schemas(name=false)
```

Comment:
Why? `generate_swagger_schemas ` can be called without any arguments :https://github.com/slate-studio/mongoid-openapi/blob/4e77a64c3ae503bb5674128fbf3e71794e988e42/lib/swagger/generator.rb#L20. Without `=false` will be error "ArgumentError (wrong number of arguments (given 0, expected 1)):"